src/main/java/axoloti/IPlatform.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (C) 2013 - 2016 Johannes Taelman
+ *
+ * This file is part of Axoloti.
+ *
+ * Axoloti is free software: you can redistribute it and/or modify it under the
+ * terms of the GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Axoloti is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+ * A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * Axoloti. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @author Mark Harris
+ */
+package axoloti;
+
+import axoloti.object.AxoObject;
+
+/**
+ *
+ * @author Mark Harris
+ */
+public interface IPlatform {
```

Comment:
I think we should merge PlatformBase and IPlatform into an abstract class called Platform where these interface methods are abstract methods. We can move the concrete method and field from PlatformBase into Platform. We're never using the PlatformBase type to refer to anything. The concrete platforms can just extend Platform without needing to implement the IPlatform interface.